### ISSUE TITLE
Report portal service become inactive intermediately 

### ISSUE THREAD

**Issue by ajinkyapande** [reportportal/reportportal#969]
Hello, I am using docker-compose approach for docker installation. My server is becoming inactive in between which is leading to failing my original test job. 

docker logs reportportal/service-api:5.1.0

```2020-05-14 10:26:35.907  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: getWidgetUsingGET_1
2020-05-14 10:26:35.933  INFO 7 --- [           main] o.s.s.quartz.SchedulerFactoryBean        : Starting Quartz Scheduler now
2020-05-14 10:26:35.981  INFO 7 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : ClusterManager: detected 2 failed or restarted instances.
2020-05-14 10:26:35.981  INFO 7 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : ClusterManager: Scanning for instance "api:3a549837d2138358ea9d55c6d782a92d"'s failed in-progress jobs.
2020-05-14 10:26:35.983  INFO 7 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : ClusterManager: Scanning for instance "api:38415490d2aba5331b44461f4ecec60b"'s failed in-progress jobs.
2020-05-14 10:26:35.985  INFO 7 --- [           main] org.quartz.core.QuartzScheduler          : Scheduler reportportal_$_api:8b4d8fb1af8dfc3c7523f0727a1ebcb9 started.
2020-05-14 10:26:36.445  INFO 7 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8585 (http) with context path ''
2020-05-14 10:26:36.454  INFO 7 --- [           main] c.e.t.r.core.configs.ReportPortalApp     : Started ReportPortalApp in 41.055 seconds (JVM running for 44.863)
2020-05-14 10:26:36.536 ERROR 7 --- [           main] c.e.t.r.f.d.minio.MinioDataStore         : Unable to find file 'keystore/migration'

io.minio.errors.ErrorResponseException: The specified key does not exist.
        at io.minio.MinioClient.executeReq(MinioClient.java:1220)
        at io.minio.MinioClient.execute(MinioClient.java:1082)
        at io.minio.MinioClient.executeGet(MinioClient.java:1339)
        at io.minio.MinioClient.getObject(MinioClient.java:1854)
        at io.minio.MinioClient.getObject(MinioClient.java:1647)
        at com.epam.ta.reportportal.filesystem.distributed.minio.MinioDataStore.load(MinioDataStore.java:82)
        at com.epam.ta.reportportal.core.events.handler.IntegrationSecretsMigrationHandler.migrate(IntegrationSecretsMigrationHandler.java:76)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.base/java.lang.reflect.Method.invoke(Unknown Source)
        at org.springframework.context.event.ApplicationListenerMethodAdapter.doInvoke(ApplicationListenerMethodAdapter.java:305)
        at org.springframework.context.event.ApplicationListenerMethodAdapter.processEvent(ApplicationListenerMethodAdapter.java:190)
        at org.springframework.context.event.ApplicationListenerMethodAdapter.onApplicationEvent(ApplicationListenerMethodAdapter.java:153)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:403)
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:360)
        at org.springframework.boot.context.event.EventPublishingRunListener.running(EventPublishingRunListener.java:103)
        at org.springframework.boot.SpringApplicationRunListeners.running(SpringApplicationRunListeners.java:77)
2020-05-14 10:26:35.599  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: getProjectUsersUsingGET_1
2020-05-14 10:26:35.699  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: bulkUpdateUsingPUT_1
2020-05-14 10:26:35.711  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: finishTestItemUsingPUT_1
2020-05-14 10:26:35.713  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: getAttributeKeysUsingGET_1
2020-05-14 10:26:35.719  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: getAttributeValuesUsingGET_1
2020-05-14 10:26:35.769  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: getTestItemsUsingGET_1
2020-05-14 10:26:35.779  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: startChildItemUsingPOST_1
2020-05-14 10:26:35.782  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: startRootItemUsingPOST_1
2020-05-14 10:26:35.828  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: getMyselfUsingGET_1
2020-05-14 10:26:35.907  INFO 7 --- [           main] .d.s.w.r.o.CachingOperationNameGenerator : Generating unique operation named: getWidgetUsingGET_1
2020-05-14 10:26:35.933  INFO 7 --- [           main] o.s.s.quartz.SchedulerFactoryBean        : Starting Quartz Scheduler now
2020-05-14 10:26:35.981  INFO 7 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : ClusterManager: detected 2 failed or restarted instances.
2020-05-14 10:26:35.981  INFO 7 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : ClusterManager: Scanning for instance "api:3a549837d2138358ea9d55c6d782a92d"'s failed in-progress jobs.
2020-05-14 10:26:35.983  INFO 7 --- [           main] o.s.s.quartz.LocalDataSourceJobStore     : ClusterManager: Scanning for instance "api:38415490d2aba5331b44461f4ecec60b"'s failed in-progress jobs.
2020-05-14 10:26:35.985  INFO 7 --- [           main] org.quartz.core.QuartzScheduler          : Scheduler reportportal_$_api:8b4d8fb1af8dfc3c7523f0727a1ebcb9 started.
2020-05-14 10:26:36.445  INFO 7 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8585 (http) with context path ''
2020-05-14 10:26:36.454  INFO 7 --- [           main] c.e.t.r.core.configs.ReportPortalApp     : Started ReportPortalApp in 41.055 seconds (JVM running for 44.863)
2020-05-14 10:26:36.536 ERROR 7 --- [           main] c.e.t.r.f.d.minio.MinioDataStore         : Unable to find file 'keystore/migration'

io.minio.errors.ErrorResponseException: The specified key does not exist.
        at io.minio.MinioClient.executeReq(MinioClient.java:1220)
        at io.minio.MinioClient.execute(MinioClient.java:1082)
        at io.minio.MinioClient.executeGet(MinioClient.java:1339)
        at io.minio.MinioClient.getObject(MinioClient.java:1854)
        at io.minio.MinioClient.getObject(MinioClient.java:1647)
        at com.epam.ta.reportportal.filesystem.distributed.minio.MinioDataStore.load(MinioDataStore.java:82)
        at com.epam.ta.reportportal.core.events.handler.IntegrationSecretsMigrationHandler.migrate(IntegrationSecretsMigrationHandler.java:76)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.base/java.lang.reflect.Method.invoke(Unknown Source)
        at org.springframework.context.event.ApplicationListenerMethodAdapter.doInvoke(ApplicationListenerMethodAdapter.java:305)
        at org.springframework.context.event.ApplicationListenerMethodAdapter.processEvent(ApplicationListenerMethodAdapter.java:190)
        at org.springframework.context.event.ApplicationListenerMethodAdapter.onApplicationEvent(ApplicationListenerMethodAdapter.java:153)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:403)
        at io.minio.MinioClient.getObject(MinioClient.java:1647)
        at com.epam.ta.reportportal.filesystem.distributed.minio.MinioDataStore.load(MinioDataStore.java:82)
        at com.epam.ta.reportportal.core.events.handler.IntegrationSecretsMigrationHandler.migrate(IntegrationSecretsMigrationHandler.java:76)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.base/java.lang.reflect.Method.invoke(Unknown Source)
        at org.springframework.context.event.ApplicationListenerMethodAdapter.doInvoke(ApplicationListenerMethodAdapter.java:305)
        at org.springframework.context.event.ApplicationListenerMethodAdapter.processEvent(ApplicationListenerMethodAdapter.java:190)
        at org.springframework.context.event.ApplicationListenerMethodAdapter.onApplicationEvent(ApplicationListenerMethodAdapter.java:153)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
        at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:403)
        at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:360)
        at org.springframework.boot.context.event.EventPublishingRunListener.running(EventPublishingRunListener.java:103)
        at org.springframework.boot.SpringApplicationRunListeners.running(SpringApplicationRunListeners.java:77)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:330)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
        at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
        at com.epam.ta.reportportal.core.configs.ReportPortalApp.main(ReportPortalApp.java:37)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
        at java.base/java.lang.reflect.Method.invoke(Unknown Source)
        at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
        at org.springframework.boot.loader.Launcher.launch(Launcher.java:51)
        at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:52)

2020-05-14 10:26:36.537  INFO 7 --- [           main] c.e.h.IntegrationSecretsMigrationHandler : Secrets migration is not needed
2020-05-14 10:34:06.151  INFO 7 --- [nio-8585-exec-2] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-05-14 10:34:06.151  INFO 7 --- [nio-8585-exec-2] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-05-14 10:34:06.277  INFO 7 --- [nio-8585-exec-2] o.s.web.servlet.DispatcherServlet        : Completed initialization in 126 ms```

**Comment by aceme007** [comment]
Running into exact same error (stack trace) and am on 5.3.1 (I have upgraded from 5.2.2 recently)
Updated the minio container to  `RELEASE.2020-10-28T08-16-50Z` but no luck.
```
api_1            | 2020-11-03 20:15:36.138  INFO 6 --- [           main] c.e.t.r.core.configs.ReportPortalApp     : Started ReportPortalApp in 19.646 seconds (JVM running for 20.534)
api_1            | 2020-11-03 20:15:36.173 ERROR 6 --- [           main] c.e.t.r.f.d.minio.MinioDataStore         : Unable to find file 'keystore/migration'
api_1            |
api_1            | io.minio.errors.ErrorResponseException: The specified key does not exist.
api_1            | 	at io.minio.MinioClient.executeReq(MinioClient.java:1220)
api_1            | 	at io.minio.MinioClient.execute(MinioClient.java:1082)
api_1            | 	at io.minio.MinioClient.executeGet(MinioClient.java:1339)
api_1            | 	at io.minio.MinioClient.getObject(MinioClient.java:1854)
api_1            | 	at io.minio.MinioClient.getObject(MinioClient.java:1647)
api_1            | 	at com.epam.ta.reportportal.filesystem.distributed.minio.MinioDataStore.load(MinioDataStore.java:83)
api_1            | 	at com.epam.ta.reportportal.core.events.handler.IntegrationSecretsMigrationHandler.migrate(IntegrationSecretsMigrationHandler.java:77)
api_1            | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
api_1            | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
api_1            | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
api_1            | 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
api_1            | 	at org.springframework.context.event.ApplicationListenerMethodAdapter.doInvoke(ApplicationListenerMethodAdapter.java:305)
api_1            | 	at org.springframework.context.event.ApplicationListenerMethodAdapter.processEvent(ApplicationListenerMethodAdapter.java:190)
api_1            | 	at org.springframework.context.event.ApplicationListenerMethodAdapter.onApplicationEvent(ApplicationListenerMethodAdapter.java:153)
api_1            | 	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
api_1            | 	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
api_1            | 	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
api_1            | 	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:403)
api_1            | 	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:360)
api_1            | 	at org.springframework.boot.context.event.EventPublishingRunListener.running(EventPublishingRunListener.java:103)
api_1            | 	at org.springframework.boot.SpringApplicationRunListeners.running(SpringApplicationRunListeners.java:77)
api_1            | 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:330)
api_1            | 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
api_1            | 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
api_1            | 	at com.epam.ta.reportportal.core.configs.ReportPortalApp.main(ReportPortalApp.java:37)
api_1            | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
api_1            | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
api_1            | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
api_1            | 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
api_1            | 	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
api_1            | 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
api_1            | 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:51)
api_1            | 	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:52)
```

**Comment by Cyberglamdring** [comment]
Dear @aceme007, @ajinkyapande
Please, update ReportPortal to the latest version and change MinIO version to `minio/minio:RELEASE.2020-10-27T04-03-55Z:`

**Comment by aceme007** [comment]
Will monitor those changes. So far it's looking good - no exceptions.
```
root@tools:/ReportPortal# docker ps
CONTAINER ID        IMAGE                                                 COMMAND                  CREATED             STATUS                    PORTS                                                 NAMES
a9b2b714a344        reportportal/service-api:5.3.3                        "/bin/sh -c ./start.…"   20 minutes ago      Up 20 minutes             8080/tcp                                              reportportal_api_1
c62ce998a46b        reportportal/service-auto-analyzer:5.3.3              "/venv/bin/uwsgi --h…"   20 minutes ago      Up 20 minutes (healthy)   5001/tcp                                              reportportal_analyzer_1
a29f7b6ec7fa        reportportal/service-index:5.0.10                     "/service-index"         20 minutes ago      Up 20 minutes             8080/tcp                                              reportportal_index_1
6c5bc8fa62d2        postgres:12-alpine                                    "docker-entrypoint.s…"   20 minutes ago      Up 20 minutes (healthy)   5432/tcp                                              reportportal_postgres_1
dca4e71cfee6        reportportal/service-authorization:5.3.3              "/bin/sh -c ./start.…"   20 minutes ago      Up 20 minutes             8080/tcp                                              reportportal_uat_1
fb8e0479698f        docker.elastic.co/elasticsearch/elasticsearch:7.3.0   "/usr/local/bin/dock…"   20 minutes ago      Up 20 minutes (healthy)   9200/tcp, 9300/tcp                                    reportportal_elasticsearch_1
f2d89a69e041        traefik:v2.0.5                                        "/entrypoint.sh --pr…"   20 minutes ago      Up 20 minutes             80/tcp, 0.0.0.0:8080-8081->8080-8081/tcp              reportportal_gateway_1
2f5c174a7d4a        reportportal/service-ui:5.3.3                         "/service-ui"            20 minutes ago      Up 20 minutes             8080/tcp                                              reportportal_ui_1
34a7c5e355f0        minio/minio:RELEASE.2020-10-27T04-03-55Z              "/usr/bin/docker-ent…"   20 minutes ago      Up 20 minutes (healthy)   9000/tcp                                              reportportal_minio_1
f062fa33a05f        rabbitmq:3.7.16-management                            "docker-entrypoint.s…"   20 minutes ago      Up 20 minutes (healthy)   4369/tcp, 5671-5672/tcp, 15671-15672/tcp, 25672/tcp   reportportal_rabbitmq_1
```

Do we want to update the production [version](https://github.com/reportportal/reportportal/blob/master/docker-compose.yml#L226) which is on `2020-05-01T22-19-14Z` as of 5.3.3?

**Comment by snehalavhare** [comment]
I am also facing the same issue even after updating the minio version to 2020-05-01T22-19-14Z. Also updated Report Portal to the latest version. The API service keeps restarting and report portal becomes inactive.

Below are the vserion details:
reportportal/service-api:5.3.5   
reportportal/service-authorization:5.3.5   
reportportal/service-auto-analyzer:5.3.5
rabbitmq:3.7.16-management 
docker.elastic.co/elasticsearch/elasticsearch:7.3.0   
reportportal/service-index:5.0.10
postgres:12-alpine  
minio/minio:RELEASE.2020-10-27T04-03-55Z 
traefik:v2.0.5  
reportportal/service-ui:5.3.5 

Can you please help to get it resolved?

**Comment by Cyberglamdring** [comment]
@snehalavhare, provide API logs please

**Comment by LuongVinhKhang** [comment]
Hi @Cyberglamdring, 
I just cloned this repo yesterday and got this on reportportal_api_1. 
I'm using Docker 3.3.1 (63152) on macOS BigSur 
```
io.minio.errors.ErrorResponseException: The specified key does not exist.
	at io.minio.MinioClient.executeReq(MinioClient.java:1220)
	at io.minio.MinioClient.execute(MinioClient.java:1082)
	at io.minio.MinioClient.executeGet(MinioClient.java:1339)
	at io.minio.MinioClient.getObject(MinioClient.java:1854)
	at io.minio.MinioClient.getObject(MinioClient.java:1647)
	at com.epam.ta.reportportal.filesystem.distributed.minio.MinioDataStore.load(MinioDataStore.java:83)
	at com.epam.ta.reportportal.core.events.handler.IntegrationSecretsMigrationHandler.migrate(IntegrationSecretsMigrationHandler.java:77)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.doInvoke(ApplicationListenerMethodAdapter.java:305)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.processEvent(ApplicationListenerMethodAdapter.java:190)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.onApplicationEvent(ApplicationListenerMethodAdapter.java:153)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:403)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:360)
	at org.springframework.boot.context.event.EventPublishingRunListener.running(EventPublishingRunListener.java:103)
	at org.springframework.boot.SpringApplicationRunListeners.running(SpringApplicationRunListeners.java:77)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:330)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at com.epam.ta.reportportal.core.configs.ReportPortalApp.main(ReportPortalApp.java:37)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:51)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:52)

2021-04-25 04:57:54.023  INFO 8 --- [           main] c.e.h.IntegrationSecretsMigrationHandler : Secrets migration is not needed
2021-04-25 04:58:14.660  INFO 8 --- [nio-8585-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2021-04-25 04:58:14.661  INFO 8 --- [nio-8585-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2021-04-25 04:58:14.740  INFO 8 --- [nio-8585-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 79 ms
Killed
```

**Comment by odubaj** [comment]
Facing the same issue, is it already resolved ?

**Comment by Maduranga** [comment]
Any update on this? Still have the issue.

**Comment by saurabhlohia** [comment]
This seems to be an issue with migration from 5.0.0 to 5.6.3 version. I am facing the same issue as well. Any inputs on this will be helpful

**Comment by Cyberglamdring** [comment]
```bash
io.minio.errors.ErrorResponseException: The specified key does not exist.
	at io.minio.MinioClient.executeReq(MinioClient.java:1220)
	at io.minio.MinioClient.execute(MinioClient.java:1082)
	at io.minio.MinioClient.executeGet(MinioClient.java:1339)
	at io.minio.MinioClient.getObject(MinioClient.java:1854)
	at io.minio.MinioClient.getObject(MinioClient.java:1647)
	at com.epam.ta.reportportal.filesystem.distributed.minio.MinioDataStore.load(MinioDataStore.java:83)
	at com.epam.ta.reportportal.core.events.handler.IntegrationSecretsMigrationHandler.migrate(IntegrationSecretsMigrationHandler.java:77)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.doInvoke(ApplicationListenerMethodAdapter.java:305)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.processEvent(ApplicationListenerMethodAdapter.java:190)
	at org.springframework.context.event.ApplicationListenerMethodAdapter.onApplicationEvent(ApplicationListenerMethodAdapter.java:153)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:403)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:360)
	at org.springframework.boot.context.event.EventPublishingRunListener.running(EventPublishingRunListener.java:103)
	at org.springframework.boot.SpringApplicationRunListeners.running(SpringApplicationRunListeners.java:77)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:330)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
	at com.epam.ta.reportportal.core.configs.ReportPortalApp.main(ReportPortalApp.java:37)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:51)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:52)
```
This error message is fine, we will remove this false message in future releases.


@LuongVinhKhang, sorry for the very long response, in your case the problem with elasticsearch. Check the permission on the ./data/elasticsearch folder. Documentation [link](https://reportportal.io/docs/Deploy-with-Docker)

**Comment by MaryiaRymarchyk** [comment]
Hello  @odubaj , @Maduranga , @saurabhlohia  
did you managed to solve this issue ?  let us know if you still have this problem otherwise ticket will be closed in 21 days.

**Comment by sharkymcdongles** [comment]
> ```shell
> io.minio.errors.ErrorResponseException: The specified key does not exist.
> 	at io.minio.MinioClient.executeReq(MinioClient.java:1220)
> 	at io.minio.MinioClient.execute(MinioClient.java:1082)
> 	at io.minio.MinioClient.executeGet(MinioClient.java:1339)
> 	at io.minio.MinioClient.getObject(MinioClient.java:1854)
> 	at io.minio.MinioClient.getObject(MinioClient.java:1647)
> 	at com.epam.ta.reportportal.filesystem.distributed.minio.MinioDataStore.load(MinioDataStore.java:83)
> 	at com.epam.ta.reportportal.core.events.handler.IntegrationSecretsMigrationHandler.migrate(IntegrationSecretsMigrationHandler.java:77)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
> 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
> 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
> 	at org.springframework.context.event.ApplicationListenerMethodAdapter.doInvoke(ApplicationListenerMethodAdapter.java:305)
> 	at org.springframework.context.event.ApplicationListenerMethodAdapter.processEvent(ApplicationListenerMethodAdapter.java:190)
> 	at org.springframework.context.event.ApplicationListenerMethodAdapter.onApplicationEvent(ApplicationListenerMethodAdapter.java:153)
> 	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172)
> 	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165)
> 	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139)
> 	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:403)
> 	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:360)
> 	at org.springframework.boot.context.event.EventPublishingRunListener.running(EventPublishingRunListener.java:103)
> 	at org.springframework.boot.SpringApplicationRunListeners.running(SpringApplicationRunListeners.java:77)
> 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:330)
> 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1226)
> 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1215)
> 	at com.epam.ta.reportportal.core.configs.ReportPortalApp.main(ReportPortalApp.java:37)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
> 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
> 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
> 	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
> 	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
> 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
> 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:51)
> 	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:52)
> ```
> 
> This error message is fine, we will remove this false message in future releases.
> 
> @LuongVinhKhang, sorry for the very long response, in your case the problem with elasticsearch. Check the permission on the ./data/elasticsearch folder. Documentation [link](https://reportportal.io/docs/Deploy-with-Docker)

which future release? it's over a year later now and I still get this on 5.8.0. is it fixed in 5.9.0?
